projects/ui/src/lib/components/po-dynamic/po-dynamic-view/po-dynamic-view.component.spec.ts

```diff
@@ -238,6 +238,131 @@ describe('PoDynamicViewComponent:', () => {
       expect(tagElement.length).toBe(fieldsTag.length);
     });
 
+    it(`should create 'po-icon' if 'tag' and 'icon'`, () => {
+      component.fields = [
+        { property: 'cpf', label: 'CPF', tag: true, icon: 'po-icon-ok' },
+      ];
+
+      component.ngOnChanges({
+        fields: new SimpleChange(null, component.fields, true)
+      });
+
+      fixture.detectChanges();
+      expect(nativeElement.querySelector('po-tag')).toBeTruthy();
+      expect(nativeElement.querySelector('.po-icon-ok')).toBeTruthy();
+    });
+
+    it(`shouldn't create 'po-icon' if 'tag' and don't have 'icon'`, () => {
+      component.fields = [
+        { property: 'cpf', label: 'CPF', tag: true },
+      ];
+
+      component.ngOnChanges({
+        fields: new SimpleChange(null, component.fields, true)
+      });
+
+      fixture.detectChanges();
+
+      expect(nativeElement.querySelector('po-tag')).toBeTruthy();
+      expect(nativeElement.querySelector('.po-icon-ok')).toBeFalsy();
+    });
+
+    it(`should create 'po-color' if 'tag' and 'color'`, () => {
+      component.fields = [
+        { property: 'cpf', label: 'CPF', tag: true, color: 'color-07' },
+      ];
+
+      component.ngOnChanges({
+        fields: new SimpleChange(null, component.fields, true)
+      });
+
+      fixture.detectChanges();
+
+      expect(nativeElement.querySelector('po-tag')).toBeTruthy();
+      expect(nativeElement.querySelector('.po-color-07')).toBeTruthy();
+    });
+
+    it(`shouldn't create 'po-color' if 'tag' and don't have 'color'`, () => {
+      component.fields = [
+        { property: 'cpf', label: 'CPF', tag: true },
+      ];
+
+      component.ngOnChanges({
+        fields: new SimpleChange(null, component.fields, true)
+      });
+
+      fixture.detectChanges();
+
+      expect(nativeElement.querySelector('po-tag')).toBeTruthy();
+      expect(nativeElement.querySelector('.po-color-07')).toBeFalsy();
+    });
+
+    it(`should create 'po-tag-inverse' if 'tag' and 'inverse'`, () => {
+      component.fields = [
+        { property: 'cpf', label: 'CPF', tag: true, color: 'color-07', inverse: true },
+      ];
+
+      component.ngOnChanges({
+        fields: new SimpleChange(null, component.fields, true)
+      });
+
+      fixture.detectChanges();
+
+      expect(nativeElement.querySelector('po-tag')).toBeTruthy();
+      expect(nativeElement.querySelector('.po-tag-inverse')).toBeTruthy();
+      expect(nativeElement.querySelector('.po-text-color-07')).toBeTruthy();
+    });
+
+    it(`shouldn't create 'po-tag-inverse' if 'tag' and 'inverse' is false`, () => {
```

Comment:
should create 'po-tag' with a custom color and without 'inverse' if only 'tag' and 'color' contain values